cypress/unit/store-mutations-setDataDictionary.cy.js

```diff
@@ -36,40 +59,44 @@ describe("setDataDictionary", () => {
                 }
             ]
         };
+
+        columns = Object.keys(state.dataTable[0]);
+        newDataDictionary = Object.assign({}, state.dataDictionary.provided);
```

Comment:
I find this assignment confusing. Unless I misunderstand, this is the same as `newDataDictionary = state.dataDictionary.provided`, yes? There is no reactivity to worry about here.

As mentioned above, I think it's clearer if you explicitly assign the `newDataDictionary` with the payload you want to reuse across the tests and remove the `.provided` mock.

Reply:
`newDataDictionary = state.dataDictionary.provided` would provide a reference to `state.dataDictionary.provided`. I want a copy of it. In this case, however, it seems we may not be considering that `Object.assign` doesn't deep copy nested objects – I think this is something you realized during the original implementation of the annotation page.

It seems that the way to do this is JSON.parse(JSON.stringify(x));

https://code.tutsplus.com/articles/the-best-way-to-deep-copy-an-object-in-javascript--cms-39655

In any case, I now explicitly define the `newDataDictionary` variable.